tools/notification-configuration/identity-resolver/identity-resolver.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RootNamespace>identity_resolver</RootNamespace>
```

Comment:
This is not .NET naming. We should call this Identity.Resolver. 

Looks like you have been using python a little to much :)

Reply:
LOL.... ok looks like the github_codeowner_subscriber did that as well. Perhaps someone else is to blame :)

Reply:
Yes. I followed existing naming standards in the solution.